quarkus/tests/integration/src/test/java/org/keycloak/it/storage/database/BasicDatabaseTest.java

```diff
@@ -92,7 +93,8 @@ public void assertManualDbInitialization(CLIResult cliResult, RawDistRootPath ra
         var output = readKeycloakDbUpdateScript(rawDistRootPath);
 
         assertThat(output, notNullValue());
-        assertThat(output, containsString("Create Database Change Log Table"));
+        // after upgrade to >= Liquibase 4.33.0, we can remove the check with the 'history' table
+        assertThat(output, CoreMatchers.anyOf(containsString("Creating database history table with name"), containsString("Creating database changelog table with name")));
```

Comment:
Changes related to:
1. checked [statement](https://github.com/liquibase/liquibase/blob/v4.33.0/liquibase-standard/src/main/java/liquibase/changelog/StandardChangeLogHistoryService.java#L262)
2. [Log message](https://github.com/liquibase/liquibase/blob/v4.33.0/liquibase-standard/src/main/java/liquibase/changelog/StandardChangeLogHistoryService.java#L272)